### DISCUSSION TITLE
Can I get Slime-style completion somehow in Sly?

### DISCUSSION THREAD

**Discussion by pavelpenev** [joaotavora/sly#574] [Q&A] [answered]
I'm trying to switch to Sly, but I've been using slime completion since 2008 and I really don't want to change all that particular part of my muscle memory. I tried using simple completion, as I strongly dislike flex completion, but unfortunately it doesn't seem to know how to complete package names. I expect typing "alex<tab>" to complete to "alexandria:", typing <tab> again should give me completions for symbols in Alexandria. This type of "2 step" completion is something I've gotten very used to and retraining myself away from it would be a great effort for me.

Otherwise I really like Sly(good job everyone), but this is unfortunately a deal breaker for now. Is there a way I can get this kind of completion in Sly?

**Comment by pavelpenev** [comment, marked as answer]
Ok, I took a look at the code and turned out to be easier than I thought to implement https://github.com/joaotavora/sly/pull/575

**Comment by joaotavora** [comment]
Lispegistus ***@***.***> writes:

> Otherwise I really like Sly(good job everyone), but this is
> unfortunately a deal breaker for now. Is there a way I can get this
> kind of completion in Sly?

I think you can just

   (setq sly-complete-symbol-function 'sly-simple-completions)

in your .emacs.  It should do the old prefix completion thing.

João
